tools/ncbi_blast_plus/ncbi_macros.xml

```diff
@@ -352,7 +352,8 @@
     <token name="@THREADS@">-num_threads "\${GALAXY_SLOTS:-8}"</token>
     <token name="@BLAST_DB_SUBJECT@">
 #if $db_opts.db_opts_selector == "db":
-  -db "${db_opts.database.fields.path}"
+  #set $dbl = " ".join(str( $db_opts.db_type.database.fields.path ).split( ',' ))
```

Comment:
That should probably not have the ``.db_type`` bit in it which is causing the TravisCI failures:

```
NotFound: cannot find 'db_type' while searching for 'db_opts.db_type.database.fields.path'
```

We can probably do this in one line without needing to set ``$dbl`` too...